docs/reference/search/suggesters/completion-suggest.asciidoc

```diff
@@ -144,17 +146,71 @@ POST music/_suggest?pretty
 The configured weight for a suggestion is returned as `score`.
 The `text` field uses the `input` of your indexed suggestion.
 
-The basic completion suggester query supports the following two parameters:
+Suggestions are document oriented, you can specify any doc values
```

Comment:
Won't non doc-values fields work too (inverted into field data), since you are using the field data APIs?  Not sure we should allow that though ...